src/io/files.js

```diff
@@ -777,7 +780,9 @@ p5.prototype.httpGet = function () {
  *
  * @method httpPost
  * @param  {String}        path       name of the file or url to load
- * @param  {String}        [datatype] "json", "jsonp", "xml", or "text"
+ * @param  {String}        [datatype] "json", "jsonp", "xml",
+ * "text", "array", or "integer"
```

Comment:
This should not be changed, the `datatype` option here refers to the return datatype from the server